src/Containers/CadView.jsx

```diff
@@ -468,7 +482,7 @@ export default function CadView({
         height: ' calc(100vh - calc(100vh - 100%))',
         minHeight: '-webkit-fill-available',
       },
-    }}data-model-ready={modelReady}
+    }} data-model-ready={modelReady}
```

Comment:
good catch.   Please move both sx and data-model-ready onto their own lines

Reply:
I have done.